### PR TITLE
Fix/adrian rules

### PR DESCRIPTION
**Pull request by levvij**

### CODE REVIEW

File: order.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+1. Imports
+2. Combined Types (`type Handler<T> = `...)
+3. Supporting small classes (`class PreviewAngle`, bigger classes into own file)
+4. Main exported Class (`export class AngleComponent`)
+5. Default subclasses/implementations (`export class HashRouter`)
+
+## General Classes
+1. Constants (`readonly animationDuration = 2000;`)
+2. Events & Observables (`onPriceUpdate: (price: number) => void;`)
+3. Variables (`animationState: nuumber;`)
```
Comment (Eggsecuter) [inline order.md]: typo: number instead of nuumber

File: new-member.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+2. Install timesheet on iPhone
+3. Install vscode
+4. Install vscode extensions
+	- GitLens
+	- Git Graph
+	- Code Spell Checker
+	- German - Code Spell Checker
+5. Add vpn
+6. Read wiki
+7. Introduce into acryps stack
```
Comment (Eggsecuter) [inline new-member.md]: Remove the example packages as the names and number might change repeatingly

File: order.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+2. Constants
+3. Events & Observables (`onPriceUpdate: (price: number) => void;`)
+4. Referenced elements (`canvas: HTMLCanvasElement`)
+5. Variables
+6. Constructor
+7. `onload`
+8. `onunload`
+9. `render`
+10. Sub-renders (`renderGalleryItem(item: RingViewModel) ...`)
+11. Public methods
```
Comment (Eggsecuter) [inline order.md]: Would place public and private methods before render and sub-renders to clearly separate html and typescript.

Currently a component would have ts, html and then again ts...

**Review by Eggsecuter — CHANGES_REQUESTED** [review]

File: order.md (hunk 3 above)
Comment (levvij) [reply order.md]: fair point

my goal was to group the component stuff (load & render), as those will be the most important areas to edit

**Review by Eggsecuter — APPROVED** [review]

**Pull request merged**